djautotask/sync_rest.py

```diff
@@ -129,6 +135,20 @@ def get_single(self, *args, **kwargs):
     def _assign_field_data(self, instance, api_instance):
         raise NotImplementedError
 
+    def _parse_date_time(self, instance, attribute_name):
+        if getattr(instance, attribute_name):
+            setattr(instance,
+                    attribute_name,
+                    parse(getattr(instance, attribute_name))
+                    )
+        return instance
+
+    def fetch_sync_by_id(self, instance_id):
```

Comment:
This is what I get when calling this method:
```sh
In [8]: ts = TicketSynchronizer()                                                                  
                                                                                                   
In [9]: ts.fetch_sync_by_id(8885)                                                                  
[2021-04-21 15:55:34] DEBUG [djautotask.api_rest:267] Making GET request to Tickets/8885
[2021-04-21 15:55:34] ERROR [djautotask.api_rest:275] Request failed: GET Tickets/8885: Invalid URL
 'Tickets/8885': No schema supplied. Perhaps you meant http://Tickets/8885?                        
```
I think this points to the `get_ticket` method not building the correct URL.

Reply:
Since the callouts make use of `fetch_sync_by_id` they are also affected by this error. This is what I get after receiving a ticket callout:
```
djautotask.api_rest.AutotaskAPIError: Invalid URL 'Tickets/8885': No schema supplied. Perhaps you m
eant http://Tickets/8885?
```

Reply:
Thanks for fixing. Callouts work now